workshops/migrations/0012_auto_20150612_0807.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+TRANSLATE_NAMES = {
+    'Git': ['swc/git-novice'],
+    'Make': [],
```

Comment:
We'll repopulate when and as lessons ripen - otherwise, there's a 
boatload of potential topics to track.  (OOP?  Data viz?  GIS? Etc.)